tasks/beats-param-check.yml

```diff
@@ -1,9 +1,14 @@
 ---
+- name: Fail when beat is not specified or blank
+  fail:
+    msg: beat must be specified and cannot be blank e.g. filebeat
+  when: not ( beat | d(false, true) )
```

Comment:
Can you use `default` filter name instead of `d` alias in your filter?
I think that would be easier to understand.

Reply:
done